.github/actions/javascript/bumpVersion/bumpVersion.ts

```diff
@@ -47,8 +48,12 @@ if (!semanticVersionLevel || !Object.keys(versionUpdater.SEMANTIC_VERSION_LEVELS
     console.log(`Invalid input for 'SEMVER_LEVEL': ${semanticVersionLevel}`, `Defaulting to: ${semanticVersionLevel}`);
 }
 
-const {version: previousVersion} = JSON.parse(fs.readFileSync('./package.json').toString());
-const newVersion = versionUpdater.incrementVersion(previousVersion, semanticVersionLevel);
+const {version: previousVersion}: PackageJson = JSON.parse(fs.readFileSync('./package.json').toString());
+if (!previousVersion) {
+    core.setFailed('Error: Could not read package.json');
+}
+
+const newVersion = versionUpdater.incrementVersion(previousVersion ?? '', semanticVersionLevel);
```

Comment:
```suggestion
const newVersion = versionUpdater.incrementVersion(previousVersion, semanticVersionLevel);
```

`previousVersion` will be always defined given the previous condition, no?
Or remove the previous condition and keep the defaulting to `''`